File: Solutions/Python/Number_Pattern.py
class Solution:
	def numberPattern(self, N):
		# code here
        s = ""
        st = []
        for i in range(1, N+1):
            for j in range(1, i+1):
                s += str(j)

            for j in range(i-1, 0, -1):
                s += str(j)
            st.append(s)
            s = ""
            
        return st

File: Solutions/Java/Number_Pattern.java
class Solution{
    ArrayList<String> numberPattern(int N){
        // code here
        String s="";
        ArrayList<String> str = new ArrayList<String>();
        for (int i=1; i<=N; i++) {
            for (int j=1; j<=i; j++) {
                s = s + String.valueOf(j);
            }
            for (int j=i-1; j>0; j--) {
                s = s + String.valueOf(j);
            }
            str.add(s);
            s = "";
        }
        return str;
    }
}
